Cargo.toml

```diff
@@ -1,7 +1,8 @@
 [package]
 name = "strftime-ruby"
-version = "0.1.0" # remember to set `html_root_url` in `src/lib.rs`.
-authors = ["Ryan Lopopolo <rjl@hyperbo.la>"]
+# remember to set `html_root_url` in `src/lib.rs`.
+version = "0.1.0"
+authors = ["Ryan Lopopolo <rjl@hyperbo.la>", "x-hgg-x"]
```

Comment:
oh sorry one more thing I forgot: can you add your name to the copyright notices in `LICENCE` and `README.md` too?